app/controllers/katello/api/v2/content_view_puppet_modules_controller.rb

```diff
@@ -24,7 +24,10 @@ def index
     param :id, String, :desc => N_("the id of the puppet module to associate")
     param :uuid, String, :desc => N_("the uuid of the puppet module to associate")
     def create
-      params[:content_view_puppet_module][:uuid] = @puppet_module.uuid if @puppet_module
+      if @puppet_module
+        params[:content_view_puppet_module][:uuid] = @puppet_module.uuid
+        params[:uuid] = @puppet_module.uuid
```

Comment:
It looks like we already have been modifying params to make sure a uuid properly passed, I would argue that this is doing the exact same thing.